plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -137,10 +154,20 @@ var FbListInlineEdit = new Class({
 
 	checkKey: function (e) {
 		var nexttds, row, index;
+		var doTab = false;
```

Comment:
please put all variable declarations in a comma separated list (see how its done in the line above this one)